app/controllers/fhir/r4/api_controller.rb

```diff
@@ -490,12 +481,20 @@ def options
 
   private
 
+  # Handle general unkown error. Log and serve a 500
+  def handle_server_error(error)
+    Rails.logger.error(error.message)
+    error.backtrace.each { |line| Rails.logger.error line }
```

Comment:
It might be a good idea combining these into one string and make one call to the logger. Perhaps something like:
Rails.logger.error ([error.message] + error.backtrace).join('\n')

Reply:
Good call. Will definitely be more performant this way https://github.com/SaraAlert/SaraAlert/pull/707/commits/46afc9977cffe736d394ed995eb0a84b357cd3cd